pkg/controllers/cluster/cluster_controller.go

```diff
@@ -321,6 +329,38 @@ func (c *Controller) ExecutionSpaceExistForCluster(clusterName string) (bool, er
 	return true, nil
 }
 
+// Delete finalizers of work objects
+func (c *Controller) deleteFinalizerForWorks(cluster *clusterv1alpha1.Cluster, workSpace *corev1.Namespace) error {
```

Comment:
```suggestion
func (c *Controller) deleteFinalizerForWorks(workSpace *corev1.Namespace) error {
```
Unused parameter `cluster`.

Reply:
fixed